openml-lightgbm/lightgbm-provider/src/main/java/com/feedzai/openml/provider/lightgbm/LightGBMSWIG.java

```diff
@@ -88,16 +76,17 @@
      * @throws ModelLoadingException in case any LightGBM error occurs.
      */
     LightGBMSWIG(final String modelPath, final DatasetSchema schema) throws ModelLoadingException {
```

Comment:
I saw that you had provided all method and variable `private` and `public` attribute, why don't you give public to class and class constructor?

Reply:
We don't want to make this low-level helper classes visible outside the package.

I can add public to the constructor though :+1: